gcloud/bigquery/test_table.py

```diff
@@ -216,6 +216,23 @@ def test_ctor_w_schema(self):
                               schema=[full_name, age])
         self.assertEqual(table.schema, [full_name, age])
 
+    def test_query_id_property(self):
+        client = _Client(self.PROJECT)
+        dataset = _Dataset(client)
+        table = self._makeOne(self.TABLE_NAME, dataset)
+        expected_result = ''.join([self.PROJECT, ':', self.DS_NAME,
+                                   '.', self.TABLE_NAME])
```

Comment:
Wouldn't this be clearer as:

```python
expected_result = '%s:%s.%s' % (
    self.PROJECT, self.DS_NAME, self.TABLE_NAME)
```

By using a different mechanism for constructing the expected string than the MUT, it also safeguards against having the same bug in code as in tests.